validator/lib/validateTileset.js

```diff
@@ -32,6 +32,19 @@ function validateNode(root, parent, resolve) {
         var tile = node.node;
         var parent = node.parent;
 
+        if (defined(tile.content.boundingVolume)) {
+            var region = tile.content.boundingVolume.region;
```

Comment:
Check that `tile.content.boundingVolume.region` and `tile.boundingVolume.region` are defined before doing a region vs region check.